### PR TITLE
Revert "Revert "Fix umd build""

### PR DESCRIPTION
**Pull request by TrySound**
Reverts gpbl/react-day-picker#656

### CODE REVIEW

**Comment by gpbl** [comment]
Still broken, going to find the fix. You can see `DateUtils` is not exported when running [the website](https://github.com/gpbl/react-day-picker/blob/master/docs/README.md) and trying to select a range 
 in this example: http://localhost:8000/examples/selected-range.

File: src/DayPicker.js [hunk 1]
```diff
@@ -574,7 +574,3 @@ export default class DayPicker extends Component {
     );
   }
 }
-
-DayPicker.DateUtils = DateUtils;
-DayPicker.LocaleUtils = LocaleUtils;
-DayPicker.ModifiersUtils = ModifiersUtils;
```
Comment (gpbl) [inline src/DayPicker.js]: Removing these lines seems to cause the issue. Are those that were causing the UMD bug?

Comment (TrySound) [reply src/DayPicker.js]: I'm digging right now.

**Comment by gpbl** [comment]
Nope it doesn't work either. Look I'm spending too much of my free time for fixing a problem nobody complains, let's keep v7 as it is.

**Comment by TrySound** [comment]
Okay. v8 shouldn't have this problem. Its exports are transparent.

**Comment by gpbl** [comment]
Let's hope that 🙏🏽

**Pull request merged**